presto-server/pom.xml

```diff
@@ -364,6 +364,14 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
```

Comment:
What is the size before and after this change

Reply:
The total size of `presto-server` is 1856MB (uncompressed, or 1681 compressed), including 193MB of `plugin/hudi`.

Reply:
Can you please do a build before this change and after this change and report the number ?

Reply:
The size changes are

. | presto-server-0.274-SNAPSHOT/ | presto-server-0.274-SNAPSHOT.tar.gz 
---- | ---- | ----
Before | 1663MB   | 1505MB
After | 1856MB  | 1681MB
+/- | +193MB   | +176MB